data/core/tips.json

```diff
@@ -0,0 +1,176 @@
+[
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Fires can spread easily, especially with abundance of fuel." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Even without electricity ovens can be useful fire containers." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Light itself doesn't draw zombies, it's sight, noise or smell." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Moose may not be your friend." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Turnout gear protects from fire but not from overheating." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Peek around corners.  You may see your enemy before it sees you." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Cold food lasts longer.  Find a cool basement or make a root cellar." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "You died?  That's a part of the experience.  Try again with what you've learned." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Freezing food?  Put it by a fire or heat it up in a pan." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Underground cave or a basement is always cool, but never freezes." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Afraid of wild life? Yell untill it goes away.  But beware of what comes instead." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Animals and zombies are not friends.  You can take advantage of that." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Door is not the only way in.  Or out if that matters." ]
+  },
+  {
+    "type": "snippet",
+    "category": "tip",
+    "text": [ "Don't fight against the odds.  Running is no shame in survival." ]
```

Comment:
Poorly constructed sentence. Try this:
"Don't fight against the odds. There's no shame in running to survive."